### DISCUSSION TITLE
https://greendolphindance.com/2024/03/07/%E5%8F%8B%E9%93%BE%E9%A1%B5%E9%9D%A2%E7%BE%8E%E5%8C%96/

### DISCUSSION THREAD

**Discussion by giscus** [greendolphindance/greendolphindance.github.io#25] [General]
# https://greendolphindance.com/2024/03/07/%E5%8F%8B%E9%93%BE%E9%A1%B5%E9%9D%A2%E7%BE%8E%E5%8C%96/

Back to the rough ground!

https://greendolphindance.com/2024/03/07/%E5%8F%8B%E9%93%BE%E9%A1%B5%E9%9D%A2%E7%BE%8E%E5%8C%96/

**Comment by ouuan** [comment]
这个“每 10 分钟”指的是你待在这个页面不动 10min（，而一开始那个是每次 build 时打乱

**Reply by greendolphindance** [reply]
对的，但是事实上改成每十分钟之后，每刷新一次也会被打乱！

**Comment by ouuan** [comment]
另外这个 `sort(() => Math.random() - 0.5)` 好抽象，shuffle 不是这么写的（，可以参考 [How to randomize (shuffle) a JavaScript array? - Stack Overflow](https://stackoverflow.com/questions/2450954/how-to-randomize-shuffle-a-javascript-array)

**Reply by greendolphindance** [reply]
但是它能跑诶🥺🥺能不能就这么算了
